### PR TITLE
[JBPM-5546] Right click does not drag or bring up browser right-click functionality

### PR DESCRIPTION
**Pull request by Christopher-Chianelli**
Depends on https://github.com/ahome-it/lienzo-core/pull/176. Right click will now act like a single left click.

### CODE REVIEW

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/main/java/org/kie/workbench/common/stunner/client/widgets/presenters/canvas/AbstractCanvasViewer.java [hunk 1]
```diff
@@ -36,6 +43,24 @@
     private final WidgetWrapperView view;
 
     public AbstractCanvasViewer(final WidgetWrapperView view) {
+        view.asWidget().addDomHandler(new ContextMenuHandler() {
+
+            @Override public void onContextMenu(ContextMenuEvent event) {
+                event.preventDefault();
+                /*NativeEvent e = event.getNativeEvent();
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/main/java/org/kie/workbench/common/stunner/client/widgets/presenters/canvas/AbstractCanvasViewer.java]: I don't suppose you meant to keep this commented block!?!

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/canvas/wires/WiresCanvasViewTest.java [hunk 2]
```diff
@@ -47,6 +48,7 @@ public void setup() {
         this.canvas = new WiresCanvasView();
         this.panel = new LienzoPanel(100,
                                      100);
+        //this.panel.setDragMouseButtons(DragMouseControl.LEFT_MOUSE_ONLY);
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/canvas/wires/WiresCanvasViewTest.java]: Please remove commented code blocks.

**Comment by Christopher-Chianelli** [comment]
@manstis This doesn't infer with native "context menu" operations except on the canvas (since the issue specified to disable context menu on shapes, and the html element shapes are contained in is the canvas). All other context menus should remain intact. I removed the commented out code I used for testing. This PR probably has to wait until the next Lienzo snapshot is released (relevant changes already merged).

**Comment by romartin** [comment]
Hey @manstis  @Christopher-Chianelli 

This cannot be tested yet as actually Stunner still depends on `lienzo core:2.0.87-RELEASE`, which is a bit old. The idea is ask Dean for a new release this week, which will include @Christopher-Chianelli  and @mdproctor improvements, plus some others, and once available in maven central,  address the IP BOM upgrade. 

Is that good for everyone? :)
Thanks!

**Comment by romartin** [comment]
hey @Christopher-Chianelli @manstis 

Let you know that probably today a new lienzo version will be released. So if all goes good, I'll create the PR for upgrading the KIE/IP BOM and any other KIE projects (stunner), and then we can address again this PR. 

Also @Christopher-Chianelli , your stuff about text wrap will be in the new release AFAIK, so feel free to create a PR for enabling the text wrap in Stunner too :) Let you know that @manstis has some changes in his fork for text stuff too, so please sync with him if necessary about this text wrap stuff, although I think there'll be no conflicts.

Thanks!

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
@Christopher-Chianelli Could you please fix the conflicts?

**Comment by Christopher-Chianelli** [comment]
Resolved. Depends on #1033.

**Comment by manstis** [comment]
#1033 has a conflict @romartin needs to resolve before I can merge.

**Comment by Christopher-Chianelli** [comment]
Fixed an issue caused by the Canvas widget class I modified being removed.

**Comment by Christopher-Chianelli** [comment]
@romartin Added tests

**Comment by Christopher-Chianelli** [comment]
Resolved merge conflicts

**Pull request merged**